inc/ThirdParty/Plugins/I18n/WPML.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace WP_Rocket\ThirdParty\Plugins\I18n;
+
+use WP_Rocket\Event_Management\Subscriber_Interface;
+
+/**
+ * Subscriber for compatibility with WPML.
+ */
+class WPML implements Subscriber_Interface {
+
+
+	/**
+	 * Events for subscriber to listen to.
+	 *
+	 * @return array
+	 */
+	public static function get_subscribed_events() {
+		if ( ! defined( 'ICL_SITEPRESS_VERSION' ) ) {
+			return [];
+		}
+
+		$events = [
+			'wcml_is_cache_enabled_for_switching_currency' => '__return_true',
```

Comment:
```suggestion
			'wcml_is_cache_enabled_for_switching_currency' => 'return_true',
```

You can't use `__return_true` directly here, but you can use the above by also using the `ReturnTypesTrait` available